lib/modules/manager/tflint-plugin/readme.md

```diff
@@ -0,0 +1,3 @@
+Renovate maintains your [TFLint configuration file](https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/config.md), and updates the plugins in the file.
+
+Supports plugins from github.com only.
```

Comment:
Do we need to explain if the TFLint manager supports private/public repositories on `github.com`?

Reply:
Technically it can support either if the token has permissions (so same as any other manager). I'm not sure if TFlint has a need for private or not though

Reply:
It doesn't support plugins in private repositories at the moment (and I don't think it will ever do, AFAIU)

Reply:
Sounds good :+1: 

I'll let you check if you also want to add something like "this is a limitation of TFLint itself".

Reply:
Yeah, let's explain _why_ we don't support private repositories. 😉 

```suggestion
Supports plugins hosted in _public_ repositories on github.com.
This is because TFLint only supports _public_ repositories.
```